docs/troubleshooting.md

```diff
@@ -25,7 +25,7 @@ your `PATH` before the Windows version, otherwise this error will appear:
 
 > `/usr/bin/env: ‘python\r’: No such file or directory`
 
-### Cannot find package after run `poetry install`
+### Cannot find package after run \`poetry install\`
```

Comment:
should be changed to quotes?

Reply:
But should this one be a command?

Reply:
It's not about a command or not. It's about how it's rendered. If you don't want it to be rendered like the `text` with a grey style, use quote "text" is like native language.

Reply:
Indeed, such rendering (in a link) would cause problems in later reStructuredText generation. However, I wanted it to be presented and understood as commands so I escaped graves by adding backslashes (\). I think I'm fine with both ways and I will keep it in this way.